### DISCUSSION TITLE
How I can change the placeholder in docsearch? I want to translate 'Search' to 'Pesquisar' and 'Search the docs' too

### DISCUSSION THREAD

**Discussion by AlbertoMar7ins** [algolia/docsearch#2300] [Q&A]
Recently I implemented the docearch in my docs page, but it is in Pt Br, so I want change the search component to 'Pesquisar' but I still haven't managed it. Does anyone have an idea how I can change this?

**Comment by randombeeper** [comment]
Hello @AlbertoMar7ins, I'd be happy to help you with this but I'm realizing that we should probably move this to Discord as this area is not well monitored. Would you mind connecting with me on our Discord at https://alg.li/discord, you can find me as jamesgrayalgolia or you can put this question in the general help Q&A section! If you're not on Discord and do not want to join you can email me at james.gray@algolia.com. Thanks!
